### ISSUE TITLE
External heater broken because of optifine

### ISSUE THREAD

**Issue by 2971Riverfront** [BluSunrize/ImmersiveEngineering#5719]
### Minecraft Version

1.18.2

### Immersive Engineering Version

1.19.4-9.4.0-166

### Expected Behavior

External heater lights up, provides power

### Actual Behavior

External heater, doesnt light up even though water wheel generates power, input connected to furnace

### Steps to reproduce

1. I uninstalled optifine from mods folder
2. uninstalled and reinstalled immersive engineering
3. tried setup again

### Debug Crash Log

1

**Comment by 2971Riverfront** [comment]
sorry, version i meant was 1.19.4 not 1.18.2

**Comment by 2971Riverfront** [comment]
***VERSION IS 1.19.4, Not 1.18.2****

**Comment by mindforger** [comment]
well the issue is optifine being not supported unfortunately

**Comment by 2971Riverfront** [comment]
> well the issue is optifine being not supported unfortunately

so i uninstalled it, why isnt it working still?

**Comment by mindforger** [comment]
> "so i uninstalled it, why isnt it working still?" 

can you rephrase? is it not working even when you uninstalled?
then try again and make a manual crash report this time like requested from you in the template that you ignored :P 
if it works now, the only thing i can say is, optifine is unsupported for now

**Comment by 2971Riverfront** [comment]
Ah there was a problem with the voltage, it works now tysm!
